plugins/org.eclipse.sirius.table.ui/src/org/eclipse/sirius/table/ui/tools/internal/editor/provider/DTableColumnLabelProvider.java

```diff
@@ -225,6 +225,9 @@ protected Option<DCell> getDCell(final Object element) {
 
     /**
      * Returns the image with the given key, or <code>null</code> if not found.
+     * 
+     * @param isSelected
```

Comment:
It's only a "happy checkstyle" here, without interest. The parameter must be documented.

Reply:
Done